element-template-generator/npm/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Is the license choice already approved by someone? If you haven't done so already, I'd at least double check with a PM 🙂